components/EntropyDeployments.tsx

```diff
@@ -232,4 +232,12 @@ export const EntropyDeployments: Record<string, EntropyDeployment> = {
     network: "mainnet",
     rpc: "https://rpc.mainnet.taiko.xyz",
   },
+  "etherlink-testnet": {
+    address: "0x23f0e8FAeE7bbb405E7A7C3d60138FCfd43d7509",
+    explorer: "https://testnet-explorer.etherlink.com/$ADDRESS",
```

Comment:
This should be `https://testnet-explorer.etherlink.com/address/$ADDRESS`

Reply:
I will fix it in the next PR.